extensions/amp-analytics/0.1/test/test-visibility-impl.js

```diff
@@ -638,6 +638,66 @@ describe('amp-analytics.visibility', () => {
       });
     });
 
+    it('"hidden" trigger should work with duration condition', () => {
```

Comment:
Does it really make sense to have a hidden trigger without duration condition?

For example, what does it mean to have a spec like following?
```json
          "hidden": {
            "on": "hidden",
            "visibilitySpec": {
              "selector": "#img-2",
              "visiblePercentageMin": 10
            },
          }
```

To me, it should probably be done with a "visible" trigger instead.